aspnetcore-runtime-60/snap/hooks/install

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh -e
```

Comment:
nitpick: Can you replace `#!/bin/sh -e` with `#!/usr/bin/env sh -e` in this in all other shell files? `/usr/bin/env` is more resilient & portable in case the shell is placed in an unusual location. Also, because it introducing a new `/bin` dependency post [DEP-17 `/usr` merge](https://dep-team.pages.debian.net/deps/dep17/).

Reply:
I changed them to
```
#!/usr/bin/env bash
set -e
```
as I was getting install errors otherwise:
```
$ sudo snap install --dangerous --devmode ./dotnet-runtime-80_8.0.7_amd64.snap 
error: cannot perform the following tasks:
- Run install hook of "dotnet-runtime-80" snap if present (run hook "install": 
-----
/usr/bin/env: ‘sh -e’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
-----)
```